tests/html.js

```diff
@@ -10,7 +10,7 @@ test('Add Errors - Pass', t => {
     },
     input: {
       description: 'I am the Bar Content Type Config textarea description',
-      html: '<label for=\'{{textarea.id}}\'>{{textarea.label}}</label><textarea id=\'{{textarea.id}}\' name=\'{{textarea.name}}\' placeholder=\'{{textarea.placeholder}}\' />{{textarea.value}}</textarea>',
+      html: '<label for=\'{{textarea.id}}\'>{{textarea.label}} <mark class=\'mark-{{textarea.required}}\'>required to {{textarea.required}}</mark></label><textarea id=\'{{textarea.id}}\' name=\'{{textarea.name}}\' placeholder=\'{{textarea.placeholder}}\' />{{textarea.value}}</textarea>',
```

Comment:
Can you also add a test in the repeatable section below. We need to make sure this is working with repeatables too.